json-sql

```diff
@@ -1,17 +1,29 @@
 #!/usr/bin/env python3
 
 import decimal
+import datetime
```

Comment:
Personally, I'm a big fan of keeping lists like this alphabetical, can you please move datetime up to the top of the list?